truss/templates/trtllm-briton/src/engine.py

```diff
@@ -132,6 +135,29 @@ def load(self):
         briton_monitor_thread.start()
         self._loaded = True
 
+    def validate_input(self, prompt, model_input):
+        # Input length <= max_input_length.
+        if prompt:
```

Comment:
prompt is not the only way input is supplied. It may be provided as a set of messages. We should also handle that case.

Reply:
True. It's handled earlier so at this point it's the prompt.